schemas/context/segmentidentity.example.1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "@id": "https://data.adobe.io/entities/segmentIdentity/id123",
```

Comment:
Can we just embed the segment id namespace in the id itself? It seems like this URI approach would lend itself to that? Otherwise this URI itself is not unique since it could collide between different namespaces.

Reply:
@harleensahni - Yes, this is a good observation, we have the same pattern with Identity. Over time we will move more over. The systems we have now are not ready for URI support in this regard.